contracts/registry/src/registry.rs

```diff
@@ -304,9 +304,40 @@ impl SBTRegistry for Contract {
     #[payable]
```

Comment:
How exactly does the storage grows? we are not adding anything new we are just modifying the existing entries

Reply:
in fact, the store can grow, if `from` has other tokens from other issuers, but `to` doesn't have (is a new account).
We should add a check similar to the one in `sbt_mint`: https://github.com/alpha-fi/i-am-human/blob/master/contracts/registry/src/registry.rs#L205

Reply:
Also, would be good to test that

Reply:
DONE